src/calculateRentalCost.js

```diff
@@ -5,6 +5,15 @@
  */
 function calculateRentalCost(days) {
   // write code here
+  let result = days * 40;
+
+  if (days >= 7) {
+    result = result - 50;
+  } else if (days >= 3) {
+    result = result - 20;
```

Comment:
Similarly, use constants for the value `3` to describe the minimum number of days required for a short term rental discount, and another constant for the discount value `20`.